lib/active_admin/middleware/error_handler.rb

```diff
@@ -0,0 +1,29 @@
+module ActiveAdmin
+  module Middleware
+    class ErrorHandler
+
+      def initialize(app)
+        @app = app
+      end
+
+      def call(env)
+        @app.call(env)
+      rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?